.github/workflows/codeql.yml

```diff
@@ -2,9 +2,17 @@ name: "Code scanning - action"
 
 on:
   push:
+    branches:
+      - master
+    tags:
+      - v*
   pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - '**/*.md'
   schedule:
-    - cron: '0 19 * * 0'
+    - cron: '00 02 * * *'
```

Comment:
Can you add a comment describing when this means -- mainly because I'm too lazy to look up crontab syntax

Reply:
I was also off 12 hours, fixed!